src/components/nav-bar/index.tsx

```diff
@@ -1,157 +1,147 @@
+        <Button href="/">
           <Image
-            className="rounded-md"
             alt="logo"
             src="/cropped-MOH-Logo-768x393.png"
-            width="180"
-            height="100"
-            // style={{ marginRight: '25px' }} // Add spacing between the image and links
+            width="220"
+            height="107"
+            className="rounded-md"
           />
-        </Box>
-
-        <Box display={'flex'} justifyContent={'flex-start'} p={2}>
-          <Button
-            href="./"
-            startIcon={<HomeIcon />}
-            style={{
-              color: '#ff8a65',
-              fontSize: '25px',
-              textTransform: 'none',
-            }}
-          >
-            Home
-          </Button>
-        </Box>
-
-        <Box display={'flex'} justifyContent={'flex-start'} p={2}>
-          <Button
-            href="/donation/add"
-            startIcon={<AddCircleIcon />}
-            style={{
-              color: '#ff8a65',
-              fontSize: '25px',
-              textTransform: 'none',
-            }}
-          >
-            Add Donation
-          </Button>
-        </Box>
-
-        <Box display={'flex'} justifyContent={'flex-start'} p={2}>
-          <Button
-            href="/donation"
-            startIcon={<VolunteerActivismIcon />}
-            style={{
-              color: '#ff8a65',
-              fontSize: '25px',
-              textTransform: 'none',
-            }}
-          >
-            Donations
-          </Button>
-        </Box>
+        </Button>
+      </Box>
 
-        <Box display={'flex'} justifyContent={'flex-start'} p={2}>
+      {/* Button Groups */}
+      {sections.map((section, index) => (
+        <Box key={index} width="100%">
+          {/* Section Header */}
           <Button
-            href="/donors"
-            startIcon={<BallotIcon />}
-            style={{
-              color: '#ff8a65',
-              fontSize: '25px',
-              textTransform: 'none',
-            }}
+            onClick={() => handleToggle(section.title)}
+            startIcon={section.icon} // Add icon here
+            endIcon={
+              expandedSection === section.title ? (
+                <ExpandLess />
+              ) : (
+                <ExpandMore />
+              )
+            }
+            style={{ ...buttonStyles, fontWeight: 'bold' }}
           >
-            Donors
+            {section.title}
           </Button>
-        </Box>
 
-        <Box display={'flex'} justifyContent={'flex-start'} p={2}>
-          <Button
-            href="/donationItem"
-            startIcon={<InventoryIcon />}
-            style={{
-              color: '#ff8a65',
-              fontSize: '25px',
-              textTransform: 'none',
-            }}
+          {/* Sub-buttons (Expandable) */}
+          <Collapse
+            in={expandedSection === section.title} // Expand only the selected section
+            timeout="auto"
+            unmountOnExit
```

Comment:
- `auto` is the default value, so this can be removed
- Our navbars are quite small, let's keep them mounted on exit

Reply:
I have removed those 2 lines.